src/web/content/static/css/letterpress.css

```diff
@@ -636,6 +636,15 @@ h2.sms-h2 a:visited
     display: flex;
 }
 
+#result-published-date {
```

Comment:
This was not styled previously.

Reply:
I sometimes use spans for marking certain data in case anyone wants to parse it, like with microformats.